### PR TITLE
Update for 18.10.0

### LINKED ISSUE #11: Makefile doesn't work anymore since 18.10.0
Even after fitting in the new filename scheme, version and checksum the makefile halts in line 93, probably because of a changed ota-zip file topology. Haven't figured out what exactly happened (late night on a Sunday). The manual zip for Fairphone Open hasn't changed regarding its structure, however. After inspecting the relevant image files updating the modem_zip-generator should be worth it.

### PR DESCRIPTION
**Pull request by z3ntu**
Fixes #11

### CODE REVIEW

**Comment by stephankusch** [comment]
Works as expected now.

**Review by Roboe — CHANGES_REQUESTED** [review]
Thank you for reminding me this at Mastodon! I forgot about it, sorry.

Overall it looks good, but I don't like the idea of including a non-versioned binary for the Edify interpreter at all on the repo, :/

The general idea of the Makefile is:
  1. get the Edify interpreter from a OTA release → set ```OTA_*``` variables
  2. get the firmware images from a MANUAL release → set ```FWUPDATE_*``` variables
    - this can also be a OTA release, if it includes every firmware img. GNU Make is a clever system and won't download two identical ZIPs.
  3. pack them into a ready-to-flash ZIP file

I'm aware that latest FP Open doesn't come with an OTA release, so we could just get the Edify interpreter from a previous OTA release. There shouldn't be any issue and doing so it keeps the Makefile more mantainable.

Could you make the changes? If not, I can do it whenever I got a computer nearby, i.e. next year. Happy New Year!

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,7 @@ all: build
 
 
 build: $(FLASHABLEZIP)
+
```
Comment (Roboe) [inline Makefile:49]: I'm not sure if this line can be there. Are you sure It doesn't conflict with Make's strict syntax?

**Review by Roboe — COMMENTED** [review]
It looks quite better now! I'll accept It once I build & test it in a few days. Thank you, good work! 😊

File: Makefile (hunk 1 above)
Comment (z3ntu) [reply Makefile:49]: L48 is just declaring a rule(?) called `build` and it has a dependency `FLASHABLEZIP`.

In (now) L50 the `FLASHABLEZIP` rule gets defined.

So afaik both lines have nothing to do with each other (and I seperated it on purpose to show that)

Comment (Roboe) [reply Makefile:49]: Oh, right, it just looked dangerous in a quick view, but it's an "alias" target (what you call rule). You are right, it looks more clear that way. I'll add another in L116 for the ```release``` target.

**Review by Roboe — APPROVED** [review]
It's installing and working fine in my FP2. Well done, @z3ntu!

**Pull request merged**